pkgs/development/libraries/libpostal/default.nix

```
@@ -1,6 +1,37 @@
-{ lib, stdenv, fetchFromGitHub, autoreconfHook }:
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, withData ? false, fetchzip }:
+let
+  # languageClassifier, libpostalData and parser were not release for libpostal 1.1, using data from 1.0 instead.
+  languageClassifier = fetchzip {
+    url = "https://github.com/openvenues/libpostal/releases/download/v1.0.0/language_classifier.tar.gz";
+    sha256 = "sha256-/Gn931Nx4UDBaiFUgGqC/NJUIKQ5aZT/+OYSlcfXva8=";
```

Nowadays the following is more common and recommended:

```suggestion
    hash = "sha256-/Gn931Nx4UDBaiFUgGqC/NJUIKQ5aZT/+OYSlcfXva8=";
```

Same for the other hashes.